daprdocs/content/en/developing-applications/building-blocks/observability/app-health.md

```diff
@@ -0,0 +1,73 @@
+> Dapr's app health checks are instead concerned with not bringing more work to an application that is not currently in a condition to accept it, but that it is expected to be able to resume doing so *eventually*. Goals include not bringing more load to an application that is already overloaded, as well as being "polite" and not taking messages from the likes of queues or pub/subs when Dapr knows the application won't be able to process them. In this regard, Dapr's app health checks are "softer" and are about waiting for an application to be able to process work, rather than terminating the running process in a "hard" way.
+
+## Configuring app health checks
+
+{{% alert title="Note" color="primary" %}}
+App health checks are currently a **preview feature** and require the `AppHealthCheck` feature flag to be enabled. Please refer to the documentation for [enabling preview features]({{<ref support-preview-features>}}).
+{{% /alert %}}
+
+App health checks are enabled with the `--enable-app-health-check` CLI flag, or the `dapr.io/enable-app-health-check: true` annotation when running on Kubernetes; they are disabled by default. Adding this flag is both necessary and sufficient to enable app health checks with the default options.
+
+The full list of options are listed in this table:
+
+| CLI flags                     | Kubernetes deployment annotation    | Description | Default value |
+| ----------------------------- | ----------------------------------- | ----------- | ------------- |
+| `--enable-app-health-check`   | `dapr.io/enable-app-health-check`   | Boolean that enables the health checks | Disabled  |
+| `--app-health-check-path`     | `dapr.io/app-health-check-path`     | Path that Dapr invokes for health probes when the app channel is HTTP (this value is ignored if the app channel is using gRPC) | `/health` |
+| `--app-health-probe-interval` | `dapr.io/app-health-probe-interval` | Number of *seconds* between each health probe | `5` |
+| `--app-health-probe-timeout`  | `dapr.io/app-health-probe-timeout`  | Timeout in *milliseconds* for health probe requests | `500` |
+| `--app-health-threshold`      | `dapr.io/app-health-threshold"`     | Max number of consecutive failures before the app is considered un-healthy | `3` |
+
+> See the [full reference]({{<ref arguments-annotations-overview>}}) for Dapr arguments and annotations for all options and how to enable them.
+
+Additionally, app health checks are impacted by the protocol used for the app channel, which is configured with the `--app-protocol` flag or the `dapr.io/app-protocol` annotation; supported values are `http` (default) or `grpc`.
+
+### Health check paths
+
+When using HTTP for `app-protocol`, Dapr will perform health probes by making a HTTP call to the path specified in `app-health-check-path`, which is `/health` by default.  
```

Comment:
```suggestion
When using HTTP for `app-protocol`, Dapr will perform health probes by making an HTTP call to the path specified in `app-health-check-path`, which is `/health` by default.  
```

Reply:
Isn't it "an HTTP" because HTTP is pronounced "aitch"?

Reply:
Yep, even though HTTP technically starts with a consonant, it sounds like it starts with a vowel... it's all in the sound :)